ouroboros-consensus/src/Ouroboros/Consensus/HeaderValidation.hs

```diff
@@ -201,12 +201,13 @@ rewindHeaderState cfg p HeaderState{..} = do
     chainState' <- rewindChainState (configConsensus cfg) headerStateChain p
     return $ HeaderState {
         headerStateChain  = chainState'
-      , headerStateTips   = Seq.dropWhileR rolledBack headerStateTips
+      , headerStateTips   = Seq.dropWhileR rollBack headerStateTips
       , headerStateAnchor = headerStateAnchor
       }
   where
-    rolledBack :: AnnTip blk -> Bool
-    rolledBack t = annTipPoint t > p
+    rollBack :: AnnTip blk -> Bool
```

Comment:
The name change was accidental, but doesn't seem worth another force push :)

Reply:
Did you see https://github.com/input-output-hk/ouroboros-network/issues/1690#issuecomment-590699588?

Reply:
I did now. I've adopted it.